python/usb_console/simulation.py

```diff
@@ -87,28 +86,34 @@ def run(self):
         while step < num_steps and self.running:
             self.sim_time = self.main_state['follower']['dynamics']['time']
 
-            # Get sensor readings
+            # Step 1. Get sensor readings from simulation
             self.sensor_readings_follower = self.eng.sensor_reading(self.main_state['follower'],self.main_state['leader'], nargout=1)
             self.sensor_readings_leader = self.eng.sensor_reading(self.main_state['leader'],self.main_state['follower'], nargout=1)
-            # update dynamics
+            
+            # Step 2. Update dynamics
             self.main_state = self.eng.main_state_update(self.main_state, nargout=1)
-            # simulate flight computers
+            
+            # Step 3. Simulate flight computers
+            # Step 3.1. Use MATLAB simulation as a base
             self.computer_state_follower, self.actuator_commands_follower = \
                 self.eng.update_FC_state(self.computer_state_follower,self.sensor_readings_follower, nargout=2)
             self.computer_state_leader, self.actuator_commands_leader = \
                 self.eng.update_FC_state(self.computer_state_leader,self.sensor_readings_leader, nargout=2)
-            # command actuators
+            
+            # Step 3.2. Send inputs and read outputs from Flight Computer
+
+            # Step 3.3. Tell flight computer to run its cycle
+            self.flight_controller_leader.write_state("cycle.start", "true")
+            self.flight_controller_follower.write_state("cycle.start", "true")
```

Comment:
This is a new feature. Only if cycle.start is true does the flight software continue on to the "execute" portion of its control cycle.